autofill/autofill-api/src/main/java/com/duckduckgo/autofill/api/duckaddress/DuckAddressStatusRepository.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autofill.api.duckaddress
+
+/**
+ * Repository for Duck Address activation status
+ *
+ * Provides ability to get and set activation status for a private duck address
+ */
+interface DuckAddressStatusRepository {
```

Comment:
Does this need to be in the public API? Doesn't seem like it's used outside of the impl module so I would put it there and make sure is not accessible.

Reply:
Good call 👍